From a714d098094eef44583fd995cbb7c17902dd39aa Mon Sep 17 00:00:00 2001
From: nx111 <gd.zhangdz@gmail.com>
Date: Sat, 6 Oct 2018 09:07:54 -0600
Subject: [PATCH 1/7] klte-common: Set SDK API level for rild

* P made changes to pthread_mutex_init() that are incompatible
  with our libsec-ril libraries. In order for rild to even see a
  radio as being availble we need the legacy behavior.
* Even though our libsec-ril libraries are from 5.0, which is API
  level 21, we do not appear to require any of the other behaviors
  from API levels earlier than 27.

Change-Id: I3b3d0160236df09bdae95c5d9f10a61bf91ebc72
---
 BoardConfigCommon.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index 441fff3..2f35c1a 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -56,7 +56,8 @@ TARGET_LD_SHIM_LIBS += \
     /system/vendor/lib/hw/camera.vendor.msm8974.so|libshim_camera.so
 TARGET_PROCESS_SDK_VERSION_OVERRIDE += \
     /system/bin/mediaserver=22 \
-    /system/vendor/bin/mm-qcamera-daemon=22
+    /system/vendor/bin/mm-qcamera-daemon=22 \
+    /system/vendor/bin/hw/rild=27
 
 # Partitions
 BOARD_BOOTIMAGE_PARTITION_SIZE := 13631488
-- 
2.17.1

